protos/cerbos/cloud/store/v1/store.proto

```diff
@@ -149,12 +149,28 @@ message ReplaceFilesRequest {
   message Condition {
     int64 store_version_must_equal = 1 [(buf.validate.field).int64.gte = 0];
   }
+
+  message Files {
+    repeated File files = 1 [
+      (buf.validate.field).repeated.min_items = 1,
+      (buf.validate.field).cel = {
```

Comment:
Unfortunately it wasn't possible to validate that the total size of the files is less than the maximum uncompressed size we allow (https://github.com/bufbuild/protovalidate/issues/380).